react-web/src/components/organisms/header/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { BrowserRouter as Router, Link } from 'react-router-dom'
+
+import classNames from 'classnames'
+
+import { Button, ButtonVariant } from 'components/atoms'
+import {
+  Heading,
+  TypographyHeadingLevel,
+} from 'components/atoms/typography/heading'
+import { Label } from 'components/atoms/typography/label'
```

Comment:
The idea is to use only the Typography component and add the `variant` to define the `Label, Heading, etc`.
If you want to change something, you just need to change the Typography file.